README.md

```diff
@@ -280,7 +290,11 @@ class IterableWeakMap {
   *[Symbol.iterator]() {
     for (const ref of this.#refSet) {
       const key = ref.deref();
-      if (!key) continue;
+      if (!key) {
+        this.#refSet.delete(ref);
+        this.#finalizationGroup.unregister(ref);
+        continue;
+      }
```

Comment:
I noticed this while writing <https://github.com/microsoft/TypeScript/pull/42274>’s test case.